src/server/lib/okta/idx/challenge.ts

```diff
@@ -9,6 +9,8 @@ import {
 } from './shared';
 import { selectAuthenticationEnrollSchema } from './enroll';
```

Comment:
It's not from this PR, but these should be absolute imports.

Reply:
I'll make a separate PR to update any absolute imports!